### ISSUE TITLE
"Full screen" keyboard shortcut causes random date change in Fantastical 2

### ISSUE THREAD

**Issue by zzamboni** [eczarny/spectacle#346]
I'm seeing a weird problem with Spectacle and Fantastical 2 for Mac. Whenever I use the "Full screen" keyboard shortcut (I usually use Cmd-Alt-Ctrl-f, but I've tried assigning other keys and I get the same behavior), the date on the calendar changes to a random date! I put together a short video showing the problem, please see the URL below.

https://www.evernote.com/shard/s2/sh/9197be43-813b-471e-b6b6-218c509298d0/97974b0b852c49a1/res/b874b4f3-a378-482f-b0a2-09911c8c9bd9/fantastical2-bug.mp4

If I maximize the window using alt-click on the green button, or enter full-screen mode, the problem does not happen, it's only when I use my Spectacle keystroke.

I don't really know if this is a problem with Fantastical or Spectacle or both - this is the only application with which I've seen any sort of side-effect of maximizing the windows. I have also reported the problem to Flexibits.

Thanks!

**Comment by zzamboni** [comment]
I upgraded Fantastical to version 2.0.2 and the problem persists.

**Comment by eczarny** [comment]
Hey @zzamboni, I can't seem to see the video. From what you're telling me it sounds like somehow Fantastical and Spectacle are both trying to use the same global shortcut. What happens if you exit Fantastical, modify the shortcut for Spectacle's "Fullscreen" action, then launch Fantastical again? Does the problem persist?

**Comment by zzamboni** [comment]
Hi @eczarny, sorry about the link, here is the correct one (I've also updated it above): https://www.evernote.com/l/AAKRl75DgTtHHra2IYxQkpjQl5dLC4UsSaE

I tried what you suggest (I actually mentioned it in my original post above) - no matter what key combination I assign to "Fullscreen", I get the same side-effect in Fantastical.

A couple of days ago Flexibits replied to my report of this issue and said they are aware of this and working on a fix, so it seems it is an issue on their side. I'll post any updates here.
